resources/qml/MessageView.qml

```diff
@@ -82,6 +82,7 @@ Item {
                 id: row
 
                 property var model
+                property int buttonSize: Settings.mobileMode? 22 : 16
```

Comment:
before they should have been implicitly dependent on whatever is bigger, font size or button size. Can you maybe make this Math.max(fontMetrics.lineHeight, 16) * 1.5 or so?